sale_order_type/models/sale.py

```diff
@@ -101,13 +114,25 @@ def create(self, vals):
     def write(self, vals):
         """A sale type could have a different order sequence, so we could
         need to change it accordingly"""
+        force_company = self.env.context.get("force_company")
+        if not force_company:
+            force_company = self.env.company.id
+        default_sequence = self._default_sequence_id()
```

Comment:
Lines 117-119 force_company are included in function

Reply:
Mmm, yes. You're right